### ISSUE TITLE
Problem with Minesweeper

### ISSUE THREAD

**Issue by SaltyBear** [michaelbutler/minesweeper#19]
Hey I found your game a week or so, loved it!  No ads, plays great.  But, I tried to use it Yesterday and it won't reveal ANY numbers other than the one you click on.   Oh, and I'm talking about your live hosted version.

Did I do something wrong?   
![minesweeper](https://user-images.githubusercontent.com/61246735/226488565-9e47e047-9f67-4fba-986c-cb9c1082d30a.jpg)

**Comment by michaelbutler** [comment]
Hi @SaltyBear -- so what browser are you using? But regardless I think what you're seeing is expected. I just tried "Expert" mode myself and I think the problem is that there are TOO many mines for the given grid size. When you click a square and a "1" appears, the game isn't suppose to reveal any more squares. It only reveals multiple blank squares when you are lucky enough to click on an empty square (and the empty square has more empty squares around it).

I should probably tweak the values of Beginner, Intermediate, and Expert mode. But ALSO someone a while back made a suggestion that the very first box click should always land on an empty space, which I think is how most Minesweeper games work, and that gives you a head start of clearing at least a little area always at the beginning.

**Comment by SaltyBear** [comment]
Yup, just checked an you were right.  Let's forget I mentioned this :)
